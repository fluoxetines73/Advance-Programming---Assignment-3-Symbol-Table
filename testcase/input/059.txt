BEGIN
INSERT z number
PRINT
BEGIN
INSERT b string
INSERT d number
ASSIGN d 123
INSERT c number
END
RPRINT
PRINT
RPRINT
END
BEGIN
INSERT d string
RPRINT
LOOKUP d
ASSIGN d '1234ABC'
END
PRINT