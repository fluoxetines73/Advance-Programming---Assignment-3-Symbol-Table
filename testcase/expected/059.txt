success
z//1
success
success
success
success
z//1
z//1
z//1
success
d//1
1
success
